modules/hmmer/eslreformat/main.nf

```diff
@@ -0,0 +1,36 @@
+process HMMER_ESLREFORMAT {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda (params.enable_conda ? "bioconda::hmmer=3.3.2" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/hmmer:3.3.2--h1b792b2_1':
+        'quay.io/biocontainers/hmmer:3.3.2--h1b792b2_1' }"
+
+    input:
+    tuple val(meta), path(seqfile)
+
+    output:
+    tuple val(meta), path("*.sequences.gz"), emit: seqreformated
+    path "versions.yml"                    , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
```

Comment:
You could also add a check here to see if the user has specified the format in the `args` and if not, you could add a default. A bit like this: https://github.com/nf-core/modules/blob/b5e3d16d80abfc6aa1c86ffde8c1b507bea80f0d/modules/bcftools/convert/main.nf#L31-L35

Reply:
That's a little bit tricky as the available formats depends on whether the input file is aligned or not. I'd rather not have to guess that, as there are so many file suffixes used for various sequence formats.

Another problem with this is that the format must come second last, after all the `--` params, before the filename, so the caller needs to get the order right in the config file.

Reply:
Allright, maybe better to leave it out then. But you should maybe specify in `meta.yml` that the user should give a format in `args`

Reply:
Yes, I was looking for a place to document that, thanks. As it doesn't fit the headers in `meta.yml`, should one do that as a comment? Do you have an example of how arguments are documented in `meta.yml`?

Reply:
I don't really know of an example, but I think it's okay if you just add it to the top description. Or you could add it to the description of the output file?

Reply:
OK, done and pushed, in the hope it didn't get too long!

Reply:
Looks good!